docs/auto_instrumentation/auto_instrumentation.rst

```diff
@@ -1,7 +1,7 @@
 OpenTelemetry Python Autoinstrumentation
 ========================================
```

Comment:
```suggestion
OpenTelemetry Python Instrumentation
====================================
```

Reply:
Also rename this file and directory.

Reply:
updated. let me know if I have missed any other doc locations, I have ctrl+f'd auto_instrumentation and others and haven't found anything but :shrug:

Reply:
Thanks!